wellcad/com/_borehole.py

```diff
@@ -2001,6 +2029,72 @@ def adjust_image_brightness_and_contrast(self, log=None, prompt_user=None):
         """
         self._dispatch.AdjustImageBrightnessAndContrast(log, prompt_user)
 
+    def retinex_filter_rgb_log(self, log=None, prompt_user=None, config=None):
+        """Applies a retinex filter to the RGB log.
+
+        Note: only compatible with WellCAD version 5.7 and onwards.
+
+        Parameters
+        ----------
+        log : str or int, optional
+            Zero based index (integer) or title (string) of
+            the log to process. If not provided, a dialog box
+            displaying a list of available logs will be displayed.
+        prompt_user : bool, optional
+            Whether dialog boxes are displayed to interact with
+            the user. If set to  ``False`` the processing parameters
+            will be retrieved from the specified configuration
+            file. If no configuration file has been specified,
+            default values will be used.
+        config : str, optional
+            Path to a configuration file or a parameter string. The
+            configuration file can contain the following options:
+
+            .. code-block:: ini
+
+                Not used
+
+        Returns
+        -------
+        Log
+            The computed log.
+        """
+
+        return Log(self._dispatch.RetinexFilterRGBLog(log, prompt_user, config))
+
+    def sharpen_rgb_log(self, log=None, prompt_user=None, config=None):
+        """Sharpens the RGB log.
+
+        Note: only compatible with WellCAD version 5.7 and onwards.
+
+        Parameters
+        ----------
+        log : str or int, optional
+            Zero based index (integer) or title (string) of
+            the log to process. If not provided, a dialog box
+            displaying a list of available logs will be displayed.
+        prompt_user : bool, optional
+            Whether dialog boxes are displayed to interact with
+            the user. If set to  ``False`` the processing parameters
+            will be retrieved from the specified configuration
+            file. If no configuration file has been specified,
+            default values will be used.
+        config : str, optional
+            Path to a configuration file or a parameter string. The
```

Comment:
Best to mark this as unused at the parameter level rather than below in the INI format documentation.